javascript/javascript3/week2/main.js

```diff
@@ -0,0 +1,38 @@
+function translateOneByOne() {
+
+
+
+    moveElement(document.querySelector("li:nth-child(1)"), { x: 20, y: 300 }).then(() => {
+        console.log("Red target has been moved");
```

Comment:
it would be good to see WHERE this target has been moved to

Reply:
I do not really know how to do it, and could not figure out why it is not working.